sdc/tests/test_rolling.py

```diff
@@ -578,6 +578,85 @@ def test_impl(series, window, min_periods):
                         ref_result = test_impl(series, window, min_periods)
                         pd.testing.assert_series_equal(jit_result, ref_result)
 
+    @skip_sdc_jit('Series.rolling.cov() unsupported Series index')
+    def test_series_rolling_cov(self):
+        def test_impl(series, window, min_periods, other, ddof):
+            return series.rolling(window, min_periods).cov(other, ddof=ddof)
+
+        hpat_func = self.jit(test_impl)
+
+        all_data = [
+            list(range(5)), [1., -1., 0., 0.1, -0.1],
+            [1., np.inf, np.inf, -1., 0., np.inf, np.NINF, np.NINF],
+            [np.nan, np.inf, np.inf, np.nan, np.nan, np.nan, np.NINF, np.NZERO]
+        ]
+        for main_data, other_data in product(all_data, all_data):
+            series = pd.Series(main_data)
+            other = pd.Series(other_data)
+            for window in range(0, len(series) + 3, 2):
+                for min_periods, ddof in product(range(0, window, 2), [0, 1]):
+                    with self.subTest(series=series, other=other, window=window,
+                                      min_periods=min_periods, ddof=ddof):
+                        jit_result = hpat_func(series, window, min_periods, other, ddof)
+                        ref_result = test_impl(series, window, min_periods, other, ddof)
+                        pd.testing.assert_series_equal(jit_result, ref_result)
+
+    @skip_sdc_jit('Series.rolling.cov() unsupported Series index')
+    def test_series_rolling_cov_default(self):
+        def test_impl(series, window, min_periods):
+            return series.rolling(window, min_periods).cov()
+
+        hpat_func = self.jit(test_impl)
+
+        all_data = [
+            list(range(5)), [1., -1., 0., 0.1, -0.1],
+            [1., np.inf, np.inf, -1., 0., np.inf, np.NINF, np.NINF],
+            [np.nan, np.inf, np.inf, np.nan, np.nan, np.nan, np.NINF, np.NZERO]
+        ]
+        for data in all_data:
+            series = pd.Series(data)
+            for window in range(0, len(series) + 3, 2):
+                for min_periods in range(0, window, 2):
+                    with self.subTest(series=series, window=window,
+                                      min_periods=min_periods):
+                        jit_result = hpat_func(series, window, min_periods)
+                        ref_result = test_impl(series, window, min_periods)
+                        pd.testing.assert_series_equal(jit_result, ref_result)
+
+    @skip_sdc_jit('Series.rolling.cov() unsupported Series index')
+    @unittest.expectedFailure
+    def test_series_rolling_cov_issue_floating_point_rounding(self):
```

Comment:
Can you please add a docstring explaining what is the problem with rounding and on which side it is (Numba or Numpy)?